tests/e2e/Services/Realtime/RealtimeCustomClientTest.php

```diff
@@ -642,11 +645,15 @@ public function testChannelDatabase()
             'x-appwrite-project' => $this->getProject()['$id'],
             'x-appwrite-key' => $this->getProject()['apiKey']
         ]), [
-            'collectionId' => 'unique()',
+            'collectionId' => ID::unique(),
             'name' => 'Actors',
-            'read' => [],
-            'write' => [],
-            'permission' => 'document'
+            'permissions' => [
+                Permission::read(Role::users()),
+                Permission::create(Role::users()),
+                Permission::update(Role::users()),
+                Permission::delete(Role::users()),
```

Comment:
Yep we need at least create permission for the current user; otherwise, only admins/API key could create documents in the collection. I've updated it to only include create permission for the current user